components/browser/state/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
     testImplementation Dependencies.testing_junit
     testImplementation Dependencies.testing_robolectric
     testImplementation Dependencies.testing_mockito
+    testImplementation Dependencies.androidx_test_junit
```

Comment:
nit: Move this to line 36 so it's in alphabetical order

Reply:
OK, re-ordered.